packages/components/src/ListView/Items/Items.component.js

```diff
@@ -13,6 +13,27 @@ const listClasses = classNames(theme['tc-list-items'], 'tc-list-items');
 const itemsClasses = classNames(theme['tc-listview-items'], 'tc-listview-items');
 const itemContainer = classNames(theme['tc-item-container'], 'tc-item-container');
 
+/**
+ * Converts a string px size in a JS valid number
+ * @param {String} sizeInPixels
+ * @returns {Number}
+ */
+function pxToInt(sizeInPixels) {
+	if (!sizeInPixels) {
+		return 0;
+	}
+
+	const parsed = parseFloat(sizeInPixels.replace('px', ''));
+
+	return isNaN(parsed) ? 0 : parsed;
```

Comment:
WDYT about `return parseFloat(sizeInPixels.replace('px', '')) || 0;` ?

Reply:
```javascript
function pxToInt(sizeInPixels = 0) {
 	return parseFloat(sizeInPixels.replace('px', '')) || 0;
}
```

Reply:
If the stylesheet happens to change, `theme` properties may be undefined, so would `sizeInPixels`, `.replace` on `undefined` === 💥

Reply:
`signInPixel` has a default value in the function signature, so this should not happen.

Reply:
Didn't see it, will change the PR (has to be a string though, `.replace` does not exist on numbers)